packages/next/src/build/webpack-config.ts

```diff
@@ -532,6 +532,27 @@ function getOptimizedAliases(): { [pkg: string]: string } {
   )
 }
 
+// Alias these modules to be resolved with "module" if possible.
+function getModularizeImportAliases(packages: string[]) {
+  const aliases: { [pkg: string]: string } = {}
+  const mainFields = ['module', 'main']
```

Comment:
Might want to add a TODO here to handle packages that're only using `exports` field